src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -249,17 +249,17 @@ Command StdIO()
             var httpPortRangeOption = new Option<HttpPortRange>(
                 "--http-port-range",
                 parseArgument: result => result.Tokens.Count == 0 ? HttpPortRange.Default : ParsePortRangeOption(result),
-                description: "Specifies the range of ports to use to enable HTTP services");
+                description: Resources.StdIoHttpPortRangeOptionDescription);
 
             var httpPortOption = new Option<HttpPort>(
                 "--http-port",
-                description: "Specifies the port on which to enable HTTP services",
+                description: Resources.StdIoHttpPortOptionDescription,
                 parseArgument: result =>
                 {
                     if (result.FindResultFor(httpPortRangeOption) is { } conflictingOption)
                     {
                         var parsed = result.Parent as OptionResult;
-                        result.ErrorMessage = $"Cannot specify both {conflictingOption.Token.Value} and {parsed.Token.Value} together";
+                        result.ErrorMessage = string.Format(Resources.StdIoErrorMessageCannotSpecifyBoth, conflictingOption.Token.Value, parsed.Token.Value);
```

Comment:
A design that avoids needing to use `string.Format` at the call sites is preferable here and will reduce bugs in the future. We can discuss what that looks like.

Reply:
Replaced string.Format with Replace, but based on the comment below it seems that it should use interpolation

Reply:
Any suggestion change to fix this ?

Reply:
Fixed